src/readthedocs_assistant/commands/migrate_config.py

```diff
@@ -23,6 +24,51 @@
 # https://github.com/readthedocs/readthedocs.org/blob/2e1b121d/readthedocs/config/config.py#L59
 CONFIG_FILENAME_REGEX = r"^\.?readthedocs.ya?ml$"
 
+MESSAGE_TPL = """Howdy! :wave:
+
+I am @readthedocs-assistant and I am sending you this pull request to
+upgrade the configuration of your Read the Docs project.
+Your project will continue working whether or not you merge it,
+but I recommended you to take it into consideration.
+
+_*Note*: This tool is in beta phase. Don't hesitate to ping
+@astrojuanlu and/or @humitos if you spot any problems._
+
+The following migrators were applied:
+
+{migrators_fragment}
+---
+
+Happy documenting! :sparkles:
+"""
+
+MIGRATOR_FRAGMENT_TPL = """- `{migrator_name}`: {migrator_one_liner}
+
+{migrator_description}
+"""
+
+
+def unwrap_text(text: str) -> str:
+    lines = text.splitlines()
+    return "".join(line.strip() + " " if line else "\n\n" for line in lines)
+
+
+def build_pull_request_body(migrators: list[Migrator]) -> str:
+    migrators_fragment = "\n".join(
+        MIGRATOR_FRAGMENT_TPL.format(
+            migrator_name=migrator.__class__.__name__,
+            migrator_one_liner=migrator.__doc__.splitlines()[0].strip(),
```

Comment:
```suggestion
            migrator_one_liner=migrator.__doc__.strip().splitlines()[0].strip(),
```

Otherwise, if the title is in the second line, it breaks.